docs/getting-started-with-dynamodb.md

```diff
@@ -0,0 +1,34 @@
+# Getting Started with Scalar DB on Dynamo DB
+
+## Overview
+This document briefly explains how you can get started with Scalar DB on Dynamo DB with a simple electronic money application.
+
+## Install prerequisites
+
+Scalar DB is written in Java. So the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+        
+From here, we assume Oracle JDK 8 is properly installed in your local environment.
+
+## Configure Scalar DB
+    
+The **scalardb.properties** (getting-started/scalardb.properties) file holds the configuration for Scalar DB. You need to update `contact_points` with aws region, `username` with your aws access key id, `password` with your aws access secret key and `storage` with `dynamo`.
```

Comment:
```suggestion
The **scalardb.properties** (getting-started/scalardb.properties) file holds the configuration for Scalar DB. You need to update `contact_points` with AWS region, `username` with your AWS access key id, `password` with your AWS access secret key and `storage` with `dynamo`.
```

Reply:
Thank you for your suggestions.
Sorry for the inconvenience, I will check this kind of mistakes before creating a PR.
Document updated accordingly.